packages/analytics-js-common/src/constants/Destinations.ts

```diff
@@ -0,0 +1,167 @@
+/*
+ GENERATED - DO NOT EDIT
+ This file is generated from the templates/Destinations.ts.template file.
+ Orginal file: https://github.com/rudderlabs/rudder-integrations-config/blob/develop/generated/Destinations.ts
```

Comment:
I guess this will be fixed later once we make it automatic. So, feel free to resolve it for now.

Reply:
it is intentional that we want to depend on development as during development we need it, we don't need to wait till it is released to develop JS device mode integration.

Reply:
Do we want to merge the integrations config PR before the integration is ready? The changes will be automatically released in its next cycle. If the integration PR is lagging behind, it could create problems.

Moroever, further such changes to the destination names in a given release cycle, could lead to multiple PRs in this repo. If not attended to properly, will lead to a mess.

I'd rather suggest to regenerate this file with every production release. The new destinations that are part of the release can be configured (I guess through the options object) to be hidden by default on our dashboard.

Once the integration is released in the SDK, we can create a hotfix to only unhide the destination in the dashboard.

Let's discuss this offline if needed.

Reply:
We need to ensure there is only one PR at any given time, either by removing duplicates or updating the same PR. The idea is that once changes are merged into the `develop` branch, they can be immediately utilized in the SDK for further development. There’s no need to wait for the release process, as having the code in the SDK doesn’t cause any issues since we still control it through the Integration configuration. Merging into `develop` allows development to start right away without waiting for the release to complete.

CC: @krishna2020 your thought?